### PR TITLE
Content changes [Part IV]

### PR DESCRIPTION
**Pull request by iwootten**
### Context

Make changes to census schemas as per tab 4 of linked spreadsheet.

### How to review

Confirm the content changes are as per the linked trello card below and that tests pass.

Launcher Links:

[census_household_eng.json](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&case_type=HI&region_code=GB-ENG&url=https://gist.githubusercontent.com/iwootten/ff66f766897e3a5c397b061b55291241/raw/cbbe5e4f967db758d6e55a74384146f673ffba7e/census_household_gb_eng.json) 
[census_household_wls.json](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&case_type=HI&region_code=GB-ENG&url=https://gist.githubusercontent.com/iwootten/d3c086cccbda4260bb5e77094bf3ab0c/raw/12116f277bd6eeb226334d08936a014b80a499bc/census_household_gb_wls.json) 
[census_individual_eng.json](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&case_type=HI&region_code=GB-ENG&url=https://gist.githubusercontent.com/iwootten/60d30c8b52dc100b7a66a08896acdbe6/raw/9099279b58db3aab04dc286c8cb8e7b88314abc5/census_individual_gb_eng.json) 
[census_individual_wls.json](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&case_type=HI&region_code=GB-ENG&url=https://gist.githubusercontent.com/iwootten/0aff8c38226f262d5fa8cdbf76b98b5a/raw/cb88c869c1851fc7bc45840c900089f5852de335/census_individual_gb_wls.json)

### CODE REVIEW

**Comment by iwootten** [comment]
[[A] Content changes - part 4 (M)](https://trello.com/c/BFfLld7Z/3784-a-content-changes-part-4-m)

File: source/jsonnet/england-wales/individual/blocks/qualifications/a_level.jsonnet [hunk 1]
```diff
@@ -9,24 +9,31 @@ local proxyTitle = {
   ],
 };
 
-local englandGuidanceTitle = 'Include equivalent qualifications achieved anywhere outside England and Wales';
-local walesGuidanceTitle = 'Include equivalent qualifications achieved anywhere outside Wales and England';
+local englandQuestionDescription = 'This could be equivalent qualifications achieved anywhere outside England and Wales';
+local walesQuestionDescription = 'This could be equivalent qualifications achieved anywhere outside Wales and England';
+
+local englandGuidance = 'These are advanced-level, subject-based qualifications that are often needed to get a place at university. Students in \nEngland and Wales usually complete AS levels by the age of 17 years and A levels by the age of 18 years.\n\nIf they have achieved similar qualifications outside of England and Wales, choose the options they think are \nthe closest match.\n\nAn International Baccalaureate diploma is equivalent to three A levels.';
```
[the remark this answers is not in the document]
Comment (iwootten) [reply source/jsonnet/england-wales/individual/blocks/qualifications/a_level.jsonnet:15]: This kind of suggests using google sheets isn't an appropriate tool for the job then?

[the remark this answers is not in the document]
Comment (MebinAbraham) [reply source/jsonnet/england-wales/individual/blocks/qualifications/a_level.jsonnet:15]: Ya it's not ideal. Hard to tell difference some times also.

File: source/jsonnet/england-wales/individual/blocks/qualifications/degree.jsonnet [hunk 2]
```diff
@@ -9,22 +9,34 @@ local proxyTitle = {
...
 
-local englandGuidanceTitle = 'Include equivalent qualifications achieved anywhere outside England and Wales';
-local walesGuidanceTitle = 'Include equivalent qualifications achieved anywhere outside Wales and England';
+local englandQuestionDescription = 'This could be equivalent qualifications achieved anywhere outside England and Wales';
+local walesQuestionDescription = 'This could be equivalent qualifications achieved anywhere outside Wales and England';
+
+local englandQuestionGuidanceList = [
+  'This is any higher education qualification, achieved in the UK or another country, at level 4 or above.',
+  'Students are usually aged 18 years or over when they begin these qualifications through a college or university.',
+  'Examples include level 4 and 5 qualifications, bachelor’s degrees with or without honours, master’s degrees, PhD or other doctorates and professional qualifications, such as PGCE or chartership.',
```
Comment (MebinAbraham) [inline source/jsonnet/england-wales/individual/blocks/qualifications/degree.jsonnet:18]: These are displayed as bullet point, I think these are just meant to be paragraphs.

File: source/jsonnet/england-wales/individual/blocks/qualifications/degree.jsonnet [hunk 3]
```diff
@@ -9,22 +9,34 @@ local proxyTitle = {
...
+];
+local walesQuestionGuidanceList = [
+  'This is any higher education qualification, achieved in the UK or another country, at level 4 or above.',
+  'Students are usually aged 18 years or over when they begin these qualifications through a college or university.',
+  'Examples include level 4 and 5 qualifications, bachelor’s degrees with or without honours, master’s degrees, PhD or other doctorates and professional qualifications, such as PGCE or chartership.',
+];
```
Comment (MebinAbraham) [inline source/jsonnet/england-wales/individual/blocks/qualifications/degree.jsonnet:24]: I can't see any differences between these strings. Have I missed something?.

File: source/jsonnet/england-wales/individual/blocks/qualifications/a_level.jsonnet [hunk 4]
```diff
@@ -9,27 +9,39 @@ local proxyTitle = {
...
+local walesQuestionDescription = 'This could be equivalent qualifications achieved anywhere outside Wales and England';
+
+local englandGuidanceList = [
+  'These are advanced-level, subject-based qualifications that are often needed to get a place at university. Students in England and Wales usually complete AS levels by the age of 17 years and A levels by the age of 18 years.',
+  'If they have achieved similar qualifications outside of England and Wales, choose the options they think are the closest match. An International Baccalaureate diploma is equivalent to three A levels.',
+];
+local walesGuidanceList = [
+  'These are advanced-level, subject-based qualifications that are often needed to get a place at university. Students in Wales and England usually complete AS levels by the age of 17 years and A levels by the age of 18 years.',
+  'If you have achieved similar qualifications outside of Wales and England, choose the options you think are the closest match. An International Baccalaureate diploma is equivalent to three A levels.',
+];
```
Comment (MebinAbraham) [inline source/jsonnet/england-wales/individual/blocks/qualifications/a_level.jsonnet:22]: These are bulleted, these should be paragraphs using: 
```json
[
  {
    "description": "some string"
  },
  {
    "description": "some string"
  },
  {
    "description": "some string"
  }
]
```

Comment (MebinAbraham) [inline source/jsonnet/england-wales/individual/blocks/qualifications/a_level.jsonnet:22]: These is only 2 variations but it should be four. At the moment, its eng/wls but it should be, 
1. Eng proxy
2. Eng non proxy
3. Wls proxy
4. Wls non proxy

Comment (MebinAbraham) [reply source/jsonnet/england-wales/individual/blocks/qualifications/a_level.jsonnet:22]: This applies to more than just a levels.

File: source/jsonnet/england-wales/individual/blocks/qualifications/degree.jsonnet (hunk 2 above)
Comment (LauraC119) [reply source/jsonnet/england-wales/individual/blocks/qualifications/degree.jsonnet:18]: yes, that's right

**Pull request merged**